packages/product-components/src/components/SelectAssetModal/AssetItemNotFound.tsx

```diff
@@ -0,0 +1,74 @@
+import { Text } from '@trezor/components';
+import { FormattedMessage } from 'react-intl';
+import { SelectAssetNetworkProps, SelectAssetSearchCategoryType } from './SelectAssetModal';
+import styled from 'styled-components';
+import { spacingsPx } from '@trezor/theme';
+
+const ColumnWrapper = styled.div<{ $height: string }>`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: ${({ $height }) => $height};
+    min-height: 180px;
+`;
+
+const ParagraphWrapper = styled.div`
```

Comment:
Not 100% sure but you might use `Paragraph` component instead of `Text` + `ParagraphWrapper`

Reply:
Nice improvement! Done